bahmni-docker/.env

```diff
@@ -139,4 +139,5 @@ CRATER_ATOMFEED_DB_PASSWORD=beehyv123
 CRATER_ATOMFEED_DB_ROOT_PASSWORD=beehyv123
 CRATER_ATOMFEED_DB_HOST=crater-atomfeed-db
 CRATER_ATOMFEED_DB_PORT=3306
-CRATER_ATOMFEED_DB_NAME=crater_atomfeed
\ No newline at end of file
+CRATER_ATOMFEED_DB_NAME=crater_atomfeed
+CRATER_URL=https://demo.craterapp.com
```

Comment:
this instance of the Crater app is not maintained by us.

Reply:
Yup. it should be localhost:81